.github/workflows/ci.yml

```
@@ -0,0 +1,100 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+name: CI Checks
+
+jobs:
+  tests:
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+    name: Build and Test (${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
```

I find this a bit nicer in my workflows (I already know that it is `-latest`, no need to make the test title that much longer).

```suggestion
          - ubuntu
          - macos
          - windows
    name: Build and Test (${{ matrix.os }})
    runs-on: ${{ matrix.os }}-latest
```

---

Oh that is nicer, thanks!